### PR TITLE
Added entities and CommentService

### PR DESCRIPTION
**Pull request by aleksandr-hrankin**

### CODE REVIEW

File: src/main/java/ua/antibyte/analyzer/entity/Comment.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    @Column(name = "helpfulness_denominator")
+    private int helpfulnessDenominator;
+    private int score;
+    private LocalDateTime time;
+    private String summary;
+    @Lob
+    private String text;
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    private Product product;
+    @ManyToOne(cascade = CascadeType.PERSIST)
```
Comment (MaliukDaria) [inline src/main/java/ua/antibyte/analyzer/entity/Comment.java:35]: а нам нужен тут CascadeType.PERSIST?

File: src/main/java/ua/antibyte/analyzer/entity/Role.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+@Data
+public class Role {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "role_name")
+    @Enumerated(value = EnumType.STRING)
+    private RoleName roleName;
+
+    public Role(RoleName roleName) {
```
Comment (KseniiaMakarova) [inline src/main/java/ua/antibyte/analyzer/entity/Role.java:26]:
```suggestion
    private Role(RoleName roleName) {
```
is it supposed to be used only by `of` method?

File: src/main/java/ua/antibyte/analyzer/service/mapper/UserMapper.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    public UserMapper(RoleService roleService) {
+        this.roleService = roleService;
+    }
+
+    public User map(CommentDto commentDto) {
+        User user = new User();
+        user.setExternalId(commentDto.getUserId());
+        user.setProfileName(commentDto.getProfileName());
+        user.setPassword("1111");
+        user.setRoles(Set.of(roleService.findByName("USER")));
```
Comment (KseniiaMakarova) [inline src/main/java/ua/antibyte/analyzer/service/mapper/UserMapper.java:22]: let's move these String values to constants

**Review by KseniiaMakarova — APPROVED** [review]
nice work!) let's also create tests for service level when you add all the necessary methods there

**Pull request merged**